pandas/core/strings/accessor.py

```diff
@@ -2210,12 +2210,20 @@ def removesuffix(self, suffix: str):
         return self._wrap_result(result)
 
     @forbid_nonstring_types(["bytes"])
-    def wrap(self, width: int, **kwargs):
+    def wrap(
+        self,
+        width: int,
+        expand_tabs: bool = True,
+        replace_whitespace: bool = True,
+        drop_whitespace: bool = True,
+        break_long_words: bool = True,
+        break_on_hyphens: bool = True,
```

Comment:
You'll need to make sure you include all these parameters: https://docs.python.org/3/library/textwrap.html#textwrap.TextWrapper

Reply:
Ah gotcha, working on this

Reply:
I've updated this to take all of the same parameters as textwrap.TextWrapper